client/src/pages/Reusable.js

```diff
@@ -47,15 +47,36 @@ export default Reusable;
 const Div = styled.div`
   display: flex;
   flex-direction: column;
-  height: 80vh;
+  height: 90vh;
   justify-content: space-around;
   padding: 1rem;
-
   text-align: center;
+  background-image: url("https://images.unsplash.com/photo-1559837957-bab8edc53c85?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxzZWFyY2h8MXx8emVybyUyMHdhc3RlfGVufDB8fDB8&auto=format&fit=crop&w=700&q=60");
```

Comment:
Consider storing this picture in your project instead of fetching it from an URL (in case it is no longer available one day)